app/routes.py

```diff
@@ -1 +1,280 @@
-from flask import Blueprint
\ No newline at end of file
+import os
+from datetime import datetime
+
+import requests
+from flask import Blueprint, abort, jsonify, make_response, request
+
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+##-----------------------------------------------------------------##
+##--------------------------Helper Functions-----------------------##
+##-----------------------------------------------------------------##
+
+def validate_item(cls,id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"{cls.__name__} {id} is invalid"}, 400))
+
+    item = cls.query.get(id)
+    if not item:
+        abort(make_response({"message": f"{cls.__name__} {id} not found"}, 404))
+
+    return item
+
+def validate_new_item(cls,request_body):
+    try:  
+        if cls == Task:
+            new_item = cls(title=request_body["title"],
+                           description=request_body["description"]
+                        )
+        else:
+             new_item = cls(title=request_body["title"])
+    except:
+        abort(make_response({"details": "Invalid data"},400))
+
+    return new_item
+
+def post_to_slack(task):
+    URL = "https://slack.com/api/chat.postMessage"
+    token = os.environ.get('SLACK_TOKEN')
+    Headers = {"Authorization" : f'Bearer {token}'}
```

Comment:
I like how you used the f-string to add "Bearer " so you could retain the environment value as the exact token obtained from Slack